timemachine/cpp/src/kernels/k_rotations.cu

```diff
@@ -165,9 +165,9 @@ void __global__ k_rotate_and_translate_mols(
             local_coords[1] += translation_x;
             local_coords[2] += translation_y;
             local_coords[3] += translation_z;
-            coords_out[(idx_in_batch * num_atoms) + (i * 3) + 0] = local_coords[1];
```

Comment:
was it just wrong before?

Reply:
(guessing it passing was because `batch_size==1` previously in all of the tests)

Reply:
Correct, was just wrong. 

There is actually a test (https://github.com/proteneer/timemachine/blob/master/tests/test_rotations.py#L70), need to follow up on what is up here.

Reply:
Take that back, that test doesn't do any batching. Will add